frameworks/next-js-files.mdx

```diff
@@ -221,7 +221,7 @@ import { pinata } from "@/utils/config"
 
 export const dynamic = "force-dynamic";
 
-export async function GET() {
```

Comment:
This is the key request, not sign. We'll keep it as is here!

Reply:
Thank you for the review.
![image](https://github.com/user-attachments/assets/97655ae2-29fb-4518-bf3c-35dc4d083f35)

If this changed to POST then it should be fixed.